Labs/AI and Machine Learning/Video Indexer/Video Indexer.md

```diff
@@ -101,136 +94,134 @@ _A famous person who appears in "Microsoft in Education"_
 
-    _Video Explorer in action_ 
+    ![Video Explorer in action](Images/video-explorer.png)
 
-Try other search terms such as "AI," "awesome," and "Microsoft." The results will show at most three videos since that's all you indexed, but feel free to upload additional videos and use Video Explorer to search them as well.
-
-<a name="Summary"></a>
-## Summary ##
+    _Video Explorer in action_
 
-The [Video Indexer API](https://api-portal.videoindexer.ai/docs/services/) is a rich one that includes methods for uploading videos for indexing, searching indexed videos, retrieving and modifying transcripts, monitoring the processing state as a video is being indexed, and more. One of the more powerful methods is [Get Video Index](https://api-portal.videoindexer.ai/docs/services/operations/operations/Get-Video-Index?), which returns the indexed content of a video containing the same kind of detailed information found in the Video Indexer portal after a video is indexed.
+Try other search terms such as "AI," "awesome," and "Microsoft." The results will show at most three videos since that's all you indexed, but feel free to upload additional videos and use Video Explorer to search them as well.
 
-Video Explorer uses the [Search Videos](https://api-portal.videoindexer.ai/docs/services/operations/operations/Search-videos?) method, which is just one of more than 30 methods featured in the Video Indexer API. You could leverage additional APIs to make Video Explorer richer and more interactive. For example, you could allow users to search for people that appear in a video, or search specifically for text that is extracted via OCR. You could even use the [Get Video Player Widget URL](https://api-portal.videoindexer.ai/docs/services/operations/operations/Get-Video-Player-Widget?) method to embed a video player in the app. Feel free to use these APIs to expand Video Explorer and customize it to fit your needs, and learn more about Video Indexer in the process.
+## Summary
 
----
+The [Video Indexer API](https://api-portal.videoindexer.ai/docs/services/?WT.mc_id=academiccontent-github-cxa) is a rich one that includes methods for uploading videos for indexing, searching indexed videos, retrieving and modifying transcripts, monitoring the processing state as a video is being indexed, and more. One of the more powerful methods is [Get Video Index](https://api-portal.videoindexer.ai/docs/services/operations/operations/Get-Video-Index?WT.mc_id=academiccontent-github-cxa), which returns the indexed content of a video containing the same kind of detailed information found in the Video Indexer portal after a video is indexed.
 
-Copyright 2019 Microsoft Corporation. All rights reserved. Except where otherwise noted, these materials are licensed under the terms of the MIT License. You may use them according to the license as is most appropriate for your project. The terms of this license can be found at https://opensource.org/licenses/MIT.
```

Comment:
Did we take out the license because we have a repo-wide license? 
(also looking at the repo-wide license, we should update that too!)

Reply:
Yup - no need to have it on every lab when it's repo wide